docs/guidelines/data-visualization/colour-usage/index.md

```diff
@@ -1,3 +1,13 @@
-# Colour Usage
+# Colour usage for data visualisation
 
-This page will provide guildelines for colour usage in graphs.
\ No newline at end of file
+Colour is highly important in data visualisation. Colour is one of the first things we perceive on a screen, and can be used to direct the users’ attention. 
+
+Colour can also be used to convey information, for example by using different colours for representing different values, or for indicating something good or bad. 
+
+However, since people see colours differently (or not at all), we need to make sure the information is accessible for everyone (see [Success criteria](/guidelines/data-visualization/success-criteria/)).
+<br>
+<br>
+
+::: info
+This page will provide guildelines for colour usage in graphs. We have not yet decided on colours, nor implemented colour tokens for data visualisation in NMP.
+:::
```

Comment:
Thanks for reviewing! Good catch. I will go through it. 
And yes, we should change all docs to UK English since that has been decided. But if you want to do that on all pages at the same time we can of course do that.